jmx/src/main/java/org/jboss/as/jmx/RemotingConnectorAdd.java

```diff
@@ -47,6 +48,11 @@ private RemotingConnectorAdd() {
     }
 
     @Override
+    protected boolean requiresRuntime(OperationContext context) {
```

Comment:
This is good; it's a fix for another (unrecorded) bug.

It's analogous to:

https://github.com/wildfly/wildfly-core/blob/master/host-controller/src/main/java/org/jboss/as/host/controller/operations/HttpManagementAddHandler.java#L102

https://github.com/wildfly/wildfly-core/blob/master/host-controller/src/main/java/org/jboss/as/host/controller/operations/NativeManagementAddHandler.java#L69

For embedded HC we didn't want it visible on the network, so those two handlers added that check. But we missed that the JMX subsystem can now run on the HC process too. Its connector should have the same behavior.

If we ever allow embedded HC to not be admin-only then we'll need to look into this handler the same way we do the other two.